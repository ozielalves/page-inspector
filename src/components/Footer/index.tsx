import React from "react";
import { ReactComponent as OzielLogo } from "../../assets/logo-footer-oziel.svg";
import { Text, ContainerWrapper } from "../../styles";

const Footer = () => {
  return (
    <ContainerWrapper style={{ padding: "0 30px 0 10px" }}>
      <OzielLogo />
      <Text style={{ marginLeft: 25, textAlign: "center" }}>
        Desenvolvido por Oziel Alves
      </Text>
    </ContainerWrapper>
  );
};

export default Footer;
