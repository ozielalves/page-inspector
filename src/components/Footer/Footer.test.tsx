import renderer from "react-test-renderer";
import React from "react";
import Footer from "./index";

describe("Footer render test", () => {
  it("should render the correct Footer correctly", () => {
    const tree = renderer
      .create(<Footer/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
