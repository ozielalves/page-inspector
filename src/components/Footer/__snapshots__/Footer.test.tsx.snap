// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render test should render the correct Footer correctly 1`] = `
<div
  className="sc-jSgupP gmVoiw"
>
  <svg>
    logo-footer-oziel.svg
  </svg>
  <p
    className="sc-gsTCUz hFbaQT"
    style={
      Object {
        "marginLeft": 25,
      }
    }
  >
    Desenvolvido por Oziel Alves
  </p>
</div>
`;
